### PR TITLE
Added botname to events and get botname from config and fixed naming …

### PR DESCRIPTION
**Pull request by BrandonMurnane**
…conflicts

### CODE REVIEW

File: refocus-bdk-client.js [hunk 1]
```diff
@@ -715,7 +716,8 @@ module.exports = (config) => {
       const events = {
         log: msg,
         roomId: room,
-        actionType: type
+        actionType: type,
+        botId: botName
```
Comment (cassiodias) [inline refocus-bdk-client.js]: bot name is tricky "promoted" to botId? Is that the idea?

File: refocus-bdk-server.js [hunk 2]
```diff
@@ -1067,7 +1068,8 @@ module.exports = (config) => {
       const events = {
         log: msg,
         roomId: room,
-        actionType: type
+        actionType: type,
+        botId: botName
```
Comment (cassiodias) [inline refocus-bdk-server.js]: same question here

File: refocus-bdk-client.js [hunk 3]
```diff
@@ -528,17 +529,17 @@ module.exports = (config) => {
      *
      * @param {String} room - Id room
      * @param {String} bot - Id of bot
-     * @param {String} botName - Name of data
+     * @param {String} botDataName - Name of data
      * @param {*} value - Can already be serialized or any other data type
      * @returns {Promise} - Bot Data response
      */
-    createBotData: (room, bot, botName, value) => {
+    createBotData: (room, bot, botDataName, value) => {
```
[the remark this answers is not in the document]
Comment (BrandonMurnane) [reply refocus-bdk-client.js]: i was very confused when i saw it

File: refocus-bdk-client.js (hunk 1 above)
Comment (BrandonMurnane) [reply refocus-bdk-client.js]: its called botid but everywhere we use it we are putting botname in

File: refocus-bdk-server.js (hunk 2 above)
Comment (cassiodias) [reply refocus-bdk-server.js]: Wonder if that is not a problem for a non-imc expert... is this happening in each bot or just in the BDK?

**Review by cassiodias — APPROVED** [review]

File: refocus-bdk-server.js [hunk 4]
```diff
@@ -593,13 +594,13 @@ module.exports = (config) => {
...
-    * @param {string} botName - Contains name of bot
+    * @param {string} name - Contains name of bot
     */
-  function startHeartBeat(botName){
+  function startHeartBeat(name){
     setInterval(() => {
       const currentTimestamp = new Date();
       const requestBody = { currentTimestamp };
-      genericPost(SERVER+API+BOTS_ROUTE+'/'+botName+'/heartbeat', requestBody,
+      genericPost(SERVER+API+BOTS_ROUTE+'/'+name+'/heartbeat', requestBody,
```
Comment (ConalCosgrove) [inline refocus-bdk-server.js]: Should these be template strings rather than concatenations? Just style wise

File: refocus-bdk-server.js [hunk 5]
```diff
@@ -1121,14 +1123,14 @@ module.exports = (config) => {
...
      */
-    refocusConnect: (app, token, botName) => {
+    refocusConnect: (app, token, name) => {
       let botId = '';
       let botRoute = '/';
       const connectToken = SOCKET_TOKEN ? SOCKET_TOKEN : token;
-      if (botName) {
-        genericGet(SERVER+API+BOTS_ROUTE+'?name='+botName,
+      if (name) {
+        genericGet(SERVER+API+BOTS_ROUTE+'?name='+name,
```
Comment (ConalCosgrove) [inline refocus-bdk-server.js]: Same as above

**Comment by kriscfoster** [comment]
@BrandonMurnane think theres just some small things to look at here

File: refocus-bdk-server.js (hunk 4 above)
Comment (BrandonMurnane) [reply refocus-bdk-server.js]: i like concatenations more than template strings as only exactly what i put in is printed

File: refocus-bdk-server.js (hunk 5 above)
Comment (BrandonMurnane) [reply refocus-bdk-server.js]: same as above

**Pull request merged**